example_analysis/3.configure_phenotype_process_params.ipynb

```diff
@@ -188,29 +188,29 @@
     "### Segmentation\n",
     "\n",
     "- `DAPI_INDEX`: The index of the DAPI channel in your image stack (0-indexed). Typically 0 for the DAPI channel from sequencing.\n",
-    "- `CYTO_CHANNEL`: The index of the cytoplasmic channel in your image stack (0-indexed). Typically the channel that best outlines cell boundaries, usually base C.\n",
+    "- `CYTO_INDEX`: The index of the cytoplasmic channel in your image stack (0-indexed). Typically the channel that best outlines cell boundaries, usually base C.\n",
```

Comment:
In this case, it is not a base, so the description here is inaccurate. However, my previous comment stands also for the SBS segmentation.